tests/test_gamma.py

```diff
@@ -26,10 +26,8 @@ def test_sd():
 )
 def test_from_quantiles(x0_x1):
     x0, x1 = x0_x1
-    if x0 == x1:
-        return
-    if x0 > x1:
-        x1, x0 = x0, x1
+    assume(abs(x1 - x0) > 1e-6)
```

Comment:
I wondered why 1e-6? Would that limit concern with the tolerance that - I suppose - is used [here](https://github.com/BAMresearch/bayem/blob/1fd1800127b16f5cbb37110acc92012c2ff3a96e/bayem/distributions.py#L187)?

Reply:
The issue for the previously failing test was x0 and x1 being too close. I guess smaller tolerances would also work, but the Gamma distributions with such close quantiles are never used, I assume. Let me know, if I should add this explanation as a comment.

Reply:
I guess, some comments could be useful.